config/fxdata/rules.cfg

```diff
@@ -258,8 +288,3 @@ PosSpellAll = SPELL_ARMOUR             BILE_DEMON BUG
 PosSpellAll = SPELL_FLIGHT             DEMONSPAWN FLY
 NegSpellAll = SPELL_FREEZE             VAMPIRE SPIDER
 NegSpellAll = SPELL_SLOW               VAMPIRE DEMONSPAWN
-; Extra sacrifices - disabled for some reason.
-;MkGoodHero = AVATAR                   KNIGHT SAMURAI WIZARD GIANT MONK
-;NegUniqFunc = COSTLIER_IMPS           TUNNELLER
-;NegUniqFunc = ALL_CREATRS_VER_ANGRY   VAMPIRE HORNY
-;PosUniqFunc = ALL_CREATRS_HAPPY       TIME_MAGE FAIRY
```

Comment:
I have these extra disabled sacrifices included to document the unique functions. There is no other way for mapmakers to know about them.

Reply:
I've now listed all possibilities at the bottom

Reply:
I agree. I've added a note about referring to creature.cfg and magic.cfg for the list